tasks/preflight.yml

```diff
@@ -15,6 +15,12 @@
   when: item in node_exporter_enabled_collectors
   with_items: "{{ node_exporter_disabled_collectors }}"
 
+- name: Get systemd version
+  shell: systemctl --version | awk '$1 == "systemd" {print $2}'
+  changed_when: false
+  check_mode: false
+  register: _systemd_version
```

Comment:
Use `node_exporter_systemd_version`

Reply:
Also this task needs:
```
  tags:
    - skip_ansible_lint
```

Reply:
done, note though that [install.yml](https://github.com/cloudalchemy/ansible-node-exporter/blob/master/tasks/install.yml#L30) uses the "_underscore" naming scheme